### ISSUE TITLE
Can I use my own mesh for Stage:Edit?

### ISSUE THREAD

**Issue by WenzhaoTang** [JiejiangWu/FaceG2E#3]
Thank you for the great work!

I have my own mesh and corresponding texture. I would like to directly edit the texture using the "edit" stage without going through the second "texture generation" stage that produces latent representations. Because I am not using a 3DMM model for UV mapping, I want to know if it’s possible to skip latent generation and directly perform texture edits on my input mesh and texture.

**Comment by JiejiangWu** [comment]
Yes it is possible to perform texture edits on your own mesh and texture. 
As your own mesh's uv is different from our pretrained uv model, you can use editing SDS loss and the consistency regularization loss, and set the texdiffusion loss to zero. (w_tex = 0, w_texyuv=0) 
https://github.com/JiejiangWu/FaceG2E/blob/edec481c0ac75250117dc0fdda86b46459b8e872/demo_editing.sh#L6-L7
